src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ImageTests/UnoSamples_Tests.Image.cs

```diff
@@ -82,18 +89,18 @@ private void RunScreenShots_Image_Alignment_page(string testName)
 		{
 			Run(testName);
 
+			var picker = _app.Marked("modesPicker");
+
 			var currentModeButton = _app.Marked("currentMode");
 			_app.WaitForElement(currentModeButton);
-
-			var nextStretchButton = _app.Marked("nextStretch");
-			nextStretchButton.Tap();
+			_app.WaitForDependencyPropertyValue(currentModeButton, "Content", "00");
 
 			for (var i = 0; i < 4; i++)
 			{
+				picker.SetDependencyPropertyValue("Mode", "S" + i);
```

Comment:
```suggestion
				// Use SetDependencyProperty instead of Tap for performance on iOS/Android (Tap takes multiple seconds to complete)
				picker.SetDependencyPropertyValue("Mode", "S" + i);
```